### ISSUE TITLE
social-login.js:749 Uncaught TypeError: Super expression must either be null or a function, not undefined

### ISSUE THREAD

**Issue by Majid-Sohail** [deepakaggarwal7/react-social-login#195]
Getting error: 
social-login.js:749 Uncaught TypeError: Super expression must either be null or a function, not undefined
After creating production build in react.

**Comment by deepakaggarwal7** [comment]
Can you ensure you are using React.Component properly as suggested here in the UI component you've created to consume react-social-login.

**Comment by deepakaggarwal7** [comment]
https://stackoverflow.com/questions/30116430/reactjs-giving-error-uncaught-typeerror-super-expression-must-either-be-null-or

**Comment by Majid-Sohail** [comment]
Yes, I'm using React.Component, I did a lot of research but couldn't find any solution. I'm using this module form past one year in production and didn't face any problem but yesterday suddenly it stops working, I ended up using a different module.

**Comment by deepakaggarwal7** [comment]
Which version were you using? If you found it in v3.4.12, you can try v.3.4.13 which might fix the issue. I had upgraded webpack and heard issues of crashing nextjs Apps. I downgraded webpack and it resolved that issue. Yours could be related too.

**Comment by deepakaggarwal7** [comment]
Is your problem resolved? If yes, what was the fix? Did the last comment help?
